ocs_ci/ocs/platform_nodes.py

```
@@ -4249,3 +4262,504 @@ def disable_nodes_network_temporarily(self, nodes, duration=20):
+
+        Args:
+            nodes (list): The OCS objects of the nodes to start
+
+        """
+        logger.info(f"Starting agent hostedcluster nodes: {[n.name for n in nodes]}")
+
+        for node in nodes:
+            self._power_operation(node.name, "on")
+            logger.info(f"Successfully started node: {node.name}")
+
+    def restart_nodes(self, nodes, wait=True):
```

Ruff (0.16.1)

[warning] Boolean default positional argument in function definition

(FBT002)